@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -49,10 +50,18 @@ export interface DecodedTile {
 }
 
 /**
- * This object keeps textual data together with metadata to place it on the map.
+ * This object keeps the path of the geometry. The path is in local tile space.
```

Comment:
> The path is in local tile space

Yes, for normal text, but not for paths added in this patch - they are in world space.

Reply:
Good spotting!